etc/samples/tendrl_definitions_generic-0.1.sample.yaml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
I would suggest another PR with sample values for CreateVolume flow may be. That would make lives easier for other PRs as few of the fields might be understood well within this PR.